msteams-platform/toolkit/debug-your-Teams-app-test-tool.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Debug your Teams App Test Tool
+author: surbhigupta 
+description: In this module, learn how to debug your Teams App Test Tool and key features of Teams App Test Tool.
+ms.author: surbhigupta 
+ms.localizationpriority: high
+ms.topic: overview
+ms.date: 11/03/2023
+---
+
+# Teams App Test Tool
+
+> [!NOTE]
+>
+> Ensure the prerelease version of Teams Toolkit is installed. For more information, see [install a prerelease version](install-Teams-Toolkit.md#install-a-pre-release-version).
+
+Teams App Test Tool (Test Tool) is a component integrated into the Teams Toolkit. The tool assists users to debug, test, and refine the app design of a Teams bot application. It provides a web-based chat environment that replicates the behavior, appearance, and user experience of Microsoft Teams. 
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the Teams App Test Tool Adaptive Card.":::
+
+## Explore Teams App Test Tool
+
+### Advantages
+
+The Teams App Test Tool offers the following benefits:
+
+1. **Sandbox environment**: Sandbox environment emulates the behavior, look and user experience of Teams.
+
+1. **Tunneling**: Tunnel service is not required as it runs locally.
+
+1. **Reduce local debugging**: Microsoft 365 Developer tenant and the app uploading permissions aren't required to debug the bot application.
```

Comment:
Do you mean `Reduce account dependencies`? I don't think "reduce local debugging" make sense here.